src/core.js

```diff
@@ -524,8 +524,8 @@
           if (params.loginUrl) {
              connect.core.getPopupManager().clear(connect.MasterTopics.LOGIN_POPUP);
           }
-          connect.core.loginWindow = connect.core.getPopupManager().open(loginUrl, connect.MasterTopics.LOGIN_POPUP);
-
+          var options = params.loginOptions ? params.loginOptions : false;
```

Comment:
Because of the way you are checking this below you don't need to do this check here.
It's just assigning `false` to something already `false` if they don't supply it.
You can just pass the `params.loginOptions` directly to `open`

Reply:
Updated to remove unneeded safety check